RecoLocalCalo/HGCalRecProducers/plugins/HGCalCLUEAlgo.cc

```diff
@@ -216,33 +220,110 @@ math::XYZPoint HGCalCLUEAlgo::calculatePosition(const std::vector<int>& v, const
     return math::XYZPoint(0.f, 0.f, 0.f);
 }
 
-void HGCalCLUEAlgo::calculateLocalDensity(const HGCalLayerTiles& lt, const unsigned int layerId, float delta_c) {
+void HGCalCLUEAlgo::calculateLocalDensity(const HGCalLayerTiles& lt,
+                                          const unsigned int layerId,
+                                          float delta_c,
+                                          float delta_r) {
   auto& cellsOnLayer = cells_[layerId];
   unsigned int numberOfCells = cellsOnLayer.detid.size();
+  bool is2by2 = true;  // use 2x2 window for scitillator as default
```

Comment:
constexpr.

should this be a configurable or an argument for this method?

Reply:
Done moving it as a configurable or an argument.